apps/extension/src/components/timePicker/TimePicker.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useRef } from 'react';
+import { FormattedInputProps } from '@shared-types/components';
+import { verificateDate, formatDate, formatTime } from '@utils/pickerUtils';
+
+const TimePicker = ({ specie, value, onChange }: FormattedInputProps) => {
+  const isDeletingRef = useRef(false);
+
```

Comment:
```suggestion
export interface TimePickerProps {
  specie: 'date' | 'time';
  value: string;
  onChange: (value: string) => void;
}

const TimePicker = ({ specie, value, onChange }: TimePickerProps ) => {
```

아래 리뷰 작성한 것처럼 저희 컨벤션대로 수정해주세요!